webapp/app/home/routes.py

```diff
@@ -146,6 +147,51 @@ def zensie_query(dt_from, dt_to):
     return df
 
 
+def aranet_query(dt_from, dt_to):
+    """
+    Performs a query for Aranet T/RH sensors.
+
+    Arguments:
+        dt_from_: date range from
+        dt_to_: date range to
+    Returns:
+        df: a df with the queried data
+    """
+
+    logging.info(
+        "Calling zensie_analysis with parameters %s %s"
```

Comment:
Info print still refers to zensie

Reply:
good spot - changed that text to remove references to "zensie" and "analysis" :)